package.json

```diff
@@ -17,7 +17,7 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "cover": "jest --coverage",
-    "lint": "eslint --quiet --ext .js --ext .jsx .",
+    "lint": "eslint --quiet --ext .js --ext .jsx --ignore-pattern '/extensions/*' .",
```

Comment:
This doesnt make any sense. This rule should go into the eslintignore file. You just removed it there ;)

Reply:
We can not eslintignore attached extensions. We should ignore them on pre-commit stage